### PR TITLE
[HDRP] Update light doc with IES scripting

### PR DESCRIPTION
**Pull request by alelievr**
### Purpose of this PR
Update light doc with IES scripting

### CODE REVIEW

**Review by sebastienlagarde — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Documentation~/Light-Component.md [hunk 1]
```diff
@@ -150,15 +150,15 @@ These settings define the emissive behavior of your Light. You can set the Light
...
 | **Cookie**                | An RGB Texture that the Light projects. For example, to create silhouettes or patterned illumination for the Light. Texture shapes should be 2D for Spot and Directional Lights and Cube for Point Lights. Always import **Cookie** textures as the default texture type. This property is available for **Spot**, **Area** (Rectangular only), **Directional**, and **Point** Lights.<br />Pyramid and Box lights will use an implicit 4x4 white cookie if none is specified. |
-| **IES Profile**                   | An IES File that describe the light profile. If a cookie and an IES profile are setup, a linear average of them is used. (In case of an IES profile and a cookie used at the same time during light baking, only the cookie will be used). |
+| **IES Profile**                   | An IES File that describe the light profile. If a cookie and an IES profile are setup, a linear average of them is used. (In case of an IES profile and a cookie used at the same time during light baking, only the cookie will be used). Note that it's not possible to assign an IES file by script, you must use the cookie property with the generated IES generated textures instead. |
```
Comment (emilybrown1) [inline com.unity.render-pipelines.high-definition/Documentation~/Light-Component.md:153]:
```suggestion
| **IES Profile**                   | An IES File that describes the light profile. If a cookie and an IES profile are set up, a linear average of them is used. (If an IES profile and a cookie are used at the same time during light baking, only the cookie will be used). Note that it's not possible to assign an IES file by script. Instead, use the cookie property with the generated IES generated textures. |
```

Comment (emilybrown1) [reply com.unity.render-pipelines.high-definition/Documentation~/Light-Component.md:153]: What are the cookie and/or the IES profile used by?

**Review by emilybrown1 — CHANGES_REQUESTED** [review]

File: com.unity.render-pipelines.high-definition/Documentation~/Light-Component.md (hunk 1 above)
Comment (alelievr) [reply com.unity.render-pipelines.high-definition/Documentation~/Light-Component.md]: Thanks for the commit suggestion :) 

The cookie and IES is used by the Light in HDRP

**Review by emilybrown1 — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Documentation~/Light-Component.md [hunk 2]
```diff
@@ -150,15 +150,15 @@ These settings define the emissive behavior of your Light. You can set the Light
...
-| **Include For Ray Tracing ** | Enable the checkbox to make this Light active when the camera has the **Ray Tracing** [Frame Setting](Frame-Settings.md) enabled. This applies to both rasterization and [ray tracing](Ray-Tracing-Getting-Started.md) passes.<br />This property only appears when you enable [additional properties](More-Options.md) for this section. It is only available in Realtime or Mixed light **Mode**. |
+| **Include For Ray Tracing** | Enable the checkbox to make this Light active when the camera has the **Ray Tracing** [Frame Setting](Frame-Settings.md) enabled. This applies to both rasterization and [ray tracing](Ray-Tracing-Getting-Started.md) passes.<br />This property only appears when you enable [additional properties](More-Options.md) for this section. It is only available in Realtime or Mixed light **Mode**. |
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/Light-Component.md:161]: Removed passive voice: 

| **Include For Ray Tracing** | Enable the checkbox to make this Light active when you enable the **Ray Tracing** [Frame Setting](Frame-Settings.md) on the Camera. This applies to rasterization and [ray tracing](Ray-Tracing-Getting-Started.md) passes.<br />This property only appears when you enable [additional properties](More-Options.md) for this section. It is only available in Realtime or Mixed light **Mode**. |

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Made some language adjustments.

**Review by Vic-Cooper — APPROVED** [review]
All good!

**Pull request merged**